.github/ISSUE_TEMPLATE/508-issue.md

```diff
@@ -1,45 +1,52 @@
 ---
-name: 508 Accessibility
-about: To describe issues related to Section 508 Accessibility Standards
+name: 508 Accessibility Issue
+about: Describe an issue related to Section 508 Accessibility Standards
 labels: 508/Accessibility
-
 ---
 
-<!--Issue Title 
-[ERROR TYPE]: 
-Brief description of error. I like to use the following for the error
-type. They are a help for searching and sorting issues.
+## Instructions
+
+1. Click or press the gear icon next to the "Labels" heading on the right. Search for your team in the labels list. If your team is not listed in the Labels menu, please leave a comment after you have created this issue. The Product Support team will create a new label for your team.
+2. If this issue is being opened by an external team, please add the `triage-incident` and `product support` labels.
```

Comment:
we don't use the term "external team" anymore. Did you mean VFS Team?

Reply:
I did, will update immediately.

Reply:
And updated